### PR TITLE
update 'unoptimized imgs' helpText and link to doc

### PR DESCRIPTION
**Pull request by kaycebasques**

### CODE REVIEW

File: lighthouse-core/audits/byte-efficiency/uses-optimized-images.js [hunk 1]
```diff
@@ -42,10 +42,10 @@ class UsesOptimizedImages extends ByteEfficiencyAudit {
       name: 'uses-optimized-images',
       description: 'Optimize images',
       informative: true,
-      helpText: 'Images should be optimized to save network bytes. ' +
-        'The following images could have smaller file sizes when compressed with ' +
-        '[WebP](https://developers.google.com/speed/webp/) or JPEG at 80 quality. ' +
-        '[Learn more about image optimization](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization).',
+      helpText: 'Optimized images take less time to download and save cellular data. ' +
+        'Consider converting the following images to WebP, or lowering their JPEG quality ' +
```
Comment (ebidel) [inline lighthouse-core/audits/byte-efficiency/uses-optimized-images.js]: If the audit passes (user has optimized all images), "Consider converting the following images..." won't make sense. We should keep the helpText agnostic of a pass/fail state.

Comment (kaycebasques) [inline lighthouse-core/audits/byte-efficiency/uses-optimized-images.js]: How about we stick to Patrick's approach [here](https://github.com/GoogleChrome/lighthouse/issues/2213#issuecomment-300598841)

Describe why the audit is important, and then just a link to the doc to find out how to fix

**Comment by kaycebasques** [comment]
@patrickhulce SGTM

**Comment by brendankenny** [comment]
resolution on this now that #2216 has landed?

**Comment by patrickhulce** [comment]
> resolution on this now that #2216 has landed? 

I think we can close, but defer to @kaycebasques in case he'd still want changes.

**Pull request closed without merging**